app/components/on-site-notifications/notification-card.js

```diff
@@ -295,6 +303,93 @@ export default Component.extend(
 			});
 		},
 
+		getMessageWallPostBody(model) {
+			let wallOwner = model.get('metadata') && model.get('metadata.username');
```

Comment:
Metadata can be a string.

Reply:
It won't be for this notification.

Reply:
This has also been updated to reflect pandora change `model.get('metadata.wallOwnerName')`.